### ISSUE TITLE
fix: TS error in generateStringArray function

### ISSUE THREAD

**Issue by JoarHansson** [sirisayshello/recipe-declutter#50]
```JS
// Function to generate array of recipeInstructions
export function generateStringArray(recipeInstructions: Instruction[]) {
  return recipeInstructions.map((instruction) => instruction.text);
}
```

`Property 'text' does not exist on type 'string'.ts(2339)`

---

I haven't noticed any errors caused by this, but we should probably investigate it further

**Comment by JoarHansson** [comment]
file: `src/lib/utils.ts`

**Comment by JoarHansson** [comment]
fixed in #101
